src/frontend/app/shared/components/list/list-types/cf-security-groups/cf-security-groups-card/cf-security-groups-card.component.html

```diff
@@ -0,0 +1,40 @@
+<app-meta-card class="sec-group-card">
+  <app-meta-card-title class="sec-group-card__title">
+    <div class="sec-group-card__title-link">{{ row.entity.name }}</div>
+  </app-meta-card-title>
+  <app-meta-card-item>
+    <app-meta-card-key>Created</app-meta-card-key>
+    <app-meta-card-value>{{ row.metadata.created_at | date: 'medium' }}</app-meta-card-value>
+  </app-meta-card-item>
+  <app-meta-card-item>
+    <app-meta-card-key>Updated</app-meta-card-key>
+    <app-meta-card-value>{{ row.metadata.updated_at | date: 'medium' }}</app-meta-card-value>
+  </app-meta-card-item>
+  <app-meta-card-item>
+    <app-meta-card-key>Staging Default</app-meta-card-key>
+    <app-meta-card-value>
+      <app-boolean-indicator [isTrue]="row.entity.staging_default" type="yesNo"></app-boolean-indicator>
+    </app-meta-card-value>
+  </app-meta-card-item>
+  <app-meta-card-item>
+    <app-meta-card-key>Running Default</app-meta-card-key>
+    <app-meta-card-value>
+      <app-boolean-indicator [isTrue]="row.entity.running_default" type="yesNo"></app-boolean-indicator>
+    </app-meta-card-value>
+  </app-meta-card-item>
+  <app-meta-card-item>
+    <app-meta-card-key>Spaces</app-meta-card-key>
+    <app-meta-card-value>
+      <div *ngIf="row.entity.spaces.length === 0">None</div>
+      <div *ngIf="row.entity.spaces.length > 0">
+        <a *ngFor="let space of row.entity.spaces" [routerLink]="getSpaceUrl(space)">{{space.entity.name}}</a>
+      </div>
+    </app-meta-card-value>
+  </app-meta-card-item>
+  <app-meta-card-item>
+    <app-meta-card-key>Rules</app-meta-card-key>
+    <app-meta-card-value>
+      <app-chips [chips]="tags"></app-chips>
```

Comment:
When these wrap, they align to the left. When used in this situation the mat-chip-list-wrapper needs to switch to `mat-chip-list-wrapper`. Also need some padding between the `+x` and last tab (think this was an existing issue)